media-proxy/src/mesh/concurrency.cc

```diff
@@ -11,19 +11,87 @@ namespace mesh {
 
 namespace context {
 
-Context::Context() : ss(std::stop_source()), ch(new thread::Channel<bool>(1))
+Context::Context() : ss(std::stop_source()),
+                     parent(nullptr),
+                     cb(nullptr),
+                     ch(new thread::Channel<bool>(1)),
+                     timeout_ms(std::chrono::milliseconds(0))
 {
 }
 
+Context::Context(Context& parent) : ss(std::stop_source()),
+                                    parent(&parent),
+                                    ch(new thread::Channel<bool>(1))
```

Comment:
For `Context::Context(Context& parent)` please also initialize `timeout_ms`
`timeout_ms(std::chrono::milliseconds(0))`

Otherwise there is usage fn uninitialized value

Reply:
Fixed https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/259/commits/e1e36e2f679761baefd657ffb9310d7959cf5637